filter.lua

```diff
@@ -30,7 +30,8 @@ local blocklists = loadlists(config.blocklists)
 local function hostname(packet, offset)
 	local str = unpacker(packet, offset)
 	local request = str(0)
-	return string.match(request, "Host:%s(.-)\r\n")
+
```

Comment:
very picky and completely up to you.. but I wouldn't introduce a new line here..